.github/ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,12 @@
+### Overview
```

Comment:
I prefer the following headings and order from the Mozilla bug template:

* What did you do? (steps to reproduce) 

* What happened? (actual results)

* What should have happened? (expected results)

Reply:
I'm easy, what the rest think about this?

Reply:
Just to say mine is based on the Mozilla template described here: https://developer.mozilla.org/en-US/docs/Mozilla/QA/Bug_writing_guidelines#General_Outline_of_a_Bug_Report and usable here: https://bugzilla.mozilla.org/enter_bug.cgi#h=bugForm%7CFirefox (behind login)